src/lib.rs

```diff
@@ -416,9 +427,15 @@ pub mod pallet {
 					supersig.members.retain(|memb| !members_to_remove.contains(memb));
 					// this will never fail, because we set provider to 1 when creating the
 					// supersig
```

Comment:
What is that about ?
Should probably be removed

Reply:
yes, forgot to remove it